api/src/server/mod.rs

```diff
@@ -42,17 +49,36 @@ pub async fn run_server(pool: Pool, host: String) -> io::Result<()> {
     // Init GraphQL schema
     let gql_schema = Arc::new(create_gql_schema());
 
+    let google_client_id = env::var("OPENID_CLIENT_ID").unwrap();
+    let google_client_secret = env::var("OPENID_CLIENT_SECRET").unwrap();
```

Comment:
Should these be named `GOOGLE_CLIENT_ID` and `GOOGLE_CLIENT_SECRET` instead?

Reply:
^

Reply:
O I did that originally but the linter was sad at me, I can remove that rule but this will probably be removed when we support multiple providers so idk if its worth